pkg/apis/pipeline/v1/pipeline_validation_test.go

```diff
@@ -2629,6 +2680,188 @@ func TestContextInvalid(t *testing.T) {
 			Also(apis.ErrGeneric(`non-existent variable in "$(context.pipeline.missing-foo)"`, "value")).
 			Also(apis.ErrGeneric(`non-existent variable in "$(context.pipelineRun.missing-foo)"`, "value")).
 			Also(apis.ErrGeneric(`non-existent variable in "$(context.pipelineTask.missing-foo)"`, "value")),
+	}}
+	for _, tt := range tests {
```

Comment:
It looks like this might contain unrelated changes from #6238, can you please remove them?

Reply:
Oh I thought you wanted me to include these changes here since the #6238 PR merged already. And @jerop agreed we could add the TestContextValidMatrix() and TestContextValidMatrix() with the removed Task Params.

Reply:
Maybe the rebase didn't work quite correctly? It looks like you're adding new test classes which probably isn't what you want

Reply:
@lbernick These test cases were never added to the PR before it was merged so I thought I would add them here?! If not I can open a separate PR for this.

From chat:
would you mind addressing my review on https://github.com/tektoncd/pipeline/pull/6238 even though it got merged? the suggestion was to remove pipelinetask.params from the test cases, since the test cases are designed to just test validation of matrix.include

Reply:
Thanks, sorry, I appreciate you addressing these comments! Yes, I think a separate PR would be better, where you can use the commit message and PR title to explain the reasoning for the changes. Otherwise, it's a bit confusing to understand why these changes are bundled in. (As you can see by me confusing myself :) )

Reply:
Okay no problem. I've created a separate PR for this here: #6279